test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -179,6 +196,88 @@ public void testSnapshotSyncApplyInterrupted() throws Exception {
         }
     }
 
+    /**
+     * Validate that no data is written into the Standby's Transaction Log during replication
+     * of 5K objects.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testStandbyTransactionLogging() throws Exception {
+        final long timeout = 30;
+
+        replicationEnded.set(false);
+
+        // (1) Subscribe Client to Standby Transaction Log
+        log.debug(">>> (1) Subscribe to Transaction Stream on Standby");
+        Future<Boolean> consumerState = subscribeTransactionStream();
+
+        // (2) Snapshot and Log Entry Sync
+        log.debug(">>> (2) Start Snapshot and Log Entry Sync");
+        testEndToEndSnapshotAndLogEntrySync();
+
+        replicationEnded.set(true);
+
+        Boolean txStreamEmpty = consumerState.get(timeout, TimeUnit.SECONDS);
+        assertThat(txStreamEmpty).isTrue();
+    }
+
+    private Future<Boolean>  subscribeTransactionStream() {
+
+        ExecutorService consumer = Executors.newSingleThreadExecutor();
+        List<CorfuRuntime> consumerRts = new ArrayList<>();
+
+        // A thread that starts and consumes transaction updates via the Transaction Stream.
+        return consumer.submit(() -> {
+
+            CorfuRuntime consumerRt = CorfuRuntime.fromParameters(CorfuRuntime.CorfuRuntimeParameters
```

Comment:
slightly unrelated to this change, but do we need to create multiple runtimes?

Reply:
no, changing to only one and shutting it down...